fastlane/Constants/Constant.swift

```diff
@@ -28,6 +28,12 @@ enum Constant {
     static let appleProductionTeamId = "<#teamId#>"
     static let keychainName = "{PROJECT_NAME}_keychain"
     static let matchURL = "git@github.com:{organization}/{repo}.git"
+    static let apiKey: [String: Any] = [
+        "key_id" : EnvironmentParser.string(key: "KEY_ID"),
+        "issuer_id": EnvironmentParser.string(key: "ISSUER_ID"),
+        "key": EnvironmentParser.string(key: "KEY"),
```

Comment:
I think we have these in `Secret.appStoreKeyIdKey` 🤔 maybe we can use the same name for .ENV.

Reply:
@blyscuit I updated it at https://github.com/nimblehq/ios-templates/pull/540/commits/0525f8a2ea700a23a3a1cd06826e9f09c2f9efde 🙏